tsconfig-backuo.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
What's up with this file? Can't we use git history?

Reply:
It's no longer needed. Removed it.